react/src/components/organisms/TeaserListing/index.js

```diff
@@ -0,0 +1,169 @@
+      'stacked': teaser.stacked,
+      'side-by-side': !teaser.stacked
+    });
+    const columnCount = (teaser.contained && teaser.gridTwoColumns) ? 2 : 3;
+    const itemsClasses = classNames({
+      'ma__teaser-listing__items': true,
+      'ma__teaser-listing__2-col-grid': columnCount === 2,
+      'ma__teaser-listing__3-col-grid': columnCount === 3
+    });
+    const shownNumber = teaser.shownItems || null;
+    const shownItems = shownNumber ? teaser.items.slice(0, shownNumber) : null;
+    const invisibleItems = (shownNumber) ? teaser.items.slice(shownNumber) : [];
+
+    let teaserHeading = 2;
+    teaserHeading = parseInt((teaser.compHeading && teaser.compHeading.level) ? teaser.compHeading.level : teaserHeading) + 1;
+    teaserHeading = parseInt((teaser.sidebarHeading && teaser.sidebarHeading.level) ? teaser.sidebarHeading.level : teaserHeading) + 1;
+
+    return (
+      <section className="ma__teaser-listing">
+        <div className="ma__teaser-listing__container">
+          {(teaser.compHeading) && (<CompHeading {...teaser.compHeading} />)}
+          {(teaser.sidebarHeading) && (<SidebarHeading {...teaser.sidebarHeading} />)}
+          {(teaser.description) && (
+            <div className="ma__teaser-listing__description">
+              <Paragraph { ...teaser.description} />
+            </div>
+          )}
+          {(teaser.featuredItems) && (
+            <div className={featuredClasses}>
+              {teaser.featuredItems.map((teaser, index) => {
+                const key = `featured-listing--item-${index}`;
+                const teaserProps = {
+                  key,
+                  level: teaserHeading,
+                  ...teaser
+                };
+                return<GeneralTeaser {...teaserProps} />;
+              })}
+            </div>
+          )}
+          {(invisibleItems.length > 0) && (
+            <div>
+              <ul className={itemsClasses}>
+                {shownItems.map((teaser, o) => {
+                  const key = `teaser-listing--item-${o}`;
+                  return(
+                    <li className="ma__teaser-listing__item" key={key}>
+                      <GeneralTeaser level={teaserHeading} {...teaser} />
+                    </li>
+                  )
+                })}
+              </ul>
+              <Collapse in={this.state.open} dimension="height">
+                <div className="ma__teaser-listing__extra">
+                  <ul className={itemsClasses}>
+                    {invisibleItems.map((teaser, u) => {
+                      const key = `hidden-teaser-listing--item-${u}`;
+                      return(
+                        <li className="ma__teaser-listing__item" key={key}>
+                          <GeneralTeaser level={teaserHeading} {...teaser} />
+                        </li>
+                      );
+                    })}
+                  </ul>
+                </div>
+              </Collapse>
+              <button
```

Comment:
seems to be some odd styling issues with this button's rendering
<img width="894" alt="screen shot 2018-12-03 at 12 54 42 pm" src="https://user-images.githubusercontent.com/9359261/49392137-8f86cb00-f6fb-11e8-8b05-fb3d1b4f92f1.png">